### PR TITLE
[Cosmos DB] Fix missing --type option for deprecation redirections (#12526)

### PR DESCRIPTION
**Pull request by sinedied**
**History Notes:**  

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/cosmosdb/commands.py [hunk 1]
```diff
@@ -52,8 +52,8 @@ def load_command_table(self, _):
     with self.command_group('cosmosdb', cosmosdb_sdk, client_factory=cf_db_accounts) as g:
         g.show_command('show', 'get')
         g.command('list-keys', 'list_keys', deprecate_info=g.deprecate(redirect='cosmosdb keys list', hide=True))
-        g.command('list-read-only-keys', 'list_read_only_keys', deprecate_info=g.deprecate(redirect='cosmosdb keys list', hide=True))
-        g.command('list-connection-strings', 'list_connection_strings', table_transformer=list_connection_strings_output, deprecate_info=g.deprecate(redirect='cosmosdb keys list', hide=True))
+        g.command('list-read-only-keys', 'list_read_only_keys', deprecate_info=g.deprecate(redirect='cosmosdb keys list --read-only-keys', hide=True))
```
Comment (shurd) [inline src/azure-cli/azure/cli/command_modules/cosmosdb/commands.py]: cosmosdb keys list --type read-only-keys

**Comment by yonzhan** [comment]
add to S167

File: src/azure-cli/azure/cli/command_modules/cosmosdb/commands.py (hunk 1 above)
Comment (fengzhou-msft) [reply src/azure-cli/azure/cli/command_modules/cosmosdb/commands.py]: @sinedied can you take a look at the above comment and fix the typo?

Comment (sinedied) [reply src/azure-cli/azure/cli/command_modules/cosmosdb/commands.py]: Ooops my bad, I pushed the fix!
BTW, did you know that you can directly propose the change during reviews in GH for such things? 😉

**Pull request merged**